test-suite/js/src/test/scala/org/scalajs/testsuite/jsinterop/ScalaJSDefinedTest.scala

```diff
@@ -116,6 +116,51 @@ object ScalaJSDefinedTest extends JasmineTest {
   }
 
   @ScalaJSDefined
+  class ConstructorDefaultParam0(val foo: Int = 0) extends js.Object
```

Comment:
I recommend against using `0` as a default, since it is also the default value of the type `Int`, which might hide bugs. Use anything but `0`.